### ISSUE TITLE
PhpStorm use spaces instead of tabs after reformat .latte

### ISSUE THREAD

**Issue by Michal920920** [nette-intellij/intellij-latte#46]
Hi, from PHPstorm 2019.2 there is a problem with formatting latte templates after reformat. It's unable set tabs instead space in settings. So even the slightest edit in latte templates will cause complete reformatting. 
The problem is just in latte macros - look at the picture.
Is there some solution how to switch formatt settings back to tabs?

![Snímek z 2019-09-13 12-40-54](https://user-images.githubusercontent.com/40000589/64857041-c7745480-d623-11e9-8c43-141615709b79.png)

**Comment by podolinek** [comment]
Same here. All latte files after reformat lines replace double brackets to spaces.

**Comment by zakrava** [comment]
@JanTvrdik @matej21 
Please make tabs vs. spaces and number of spaces selectable/configurable in settings.

**Comment by rozsival** [comment]
Interesting, after upgrading to PhpStorm 2019.3 the issue is gone. Anyone else can confirm that?

**Comment by kolinger** [comment]
Not for me. Tabs are still converted to spaces (with latest PhpStorm 2019.3).

**Comment by rozsival** [comment]
@kolinger Ok, it seems like that in my case it works in combination with `.editorconfig`.

```editorconfig
[*.latte]
indent_style = tab
end_of_line = lf
trim_trailing_whitespace = true
insert_final_newline = true
```

In previous versions of PhpStorm, the EditorConfig plugin was external and I was facing this issue even with the mentioned setup. But as of 2019.3 the EditorConfig plugin was updated and it is now bundled in PhpStorm and it seems it helps. I tried it in multiple projects, `Reformat Code` now works as expected.

I understand this is not a proper solution, but for now, it might help others too.

**Comment by kolinger** [comment]
Yes, this works for me too. I placed `.editorconfig` in root directory of my projects with your contents. And after IDE restart latte is formatted properly everywhere. Thank you!

**Comment by Jigsik** [comment]
`.editorconfig` works like a charm. Thanks!

**Comment by mesour** [comment]
Fixed in 1.0.2. It can be set in code style settings for Latte in PHPStorm preferences
